src/main/webapp/components/Table.js

```diff
@@ -0,0 +1,90 @@
+import { Component } from 'react';
+import { a, hh, h, div, p, hr, small } from 'react-hyperscript-helpers';
+import React from 'react';
+import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table';
```

Comment:
nit: spacing - rest of the imports have spaces after/before the braces. Let's try to be consistent.
```suggestion
import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
```